lua/cscope_maps.lua

```diff
@@ -17,7 +17,7 @@ M.setup = function(opts)
 
 	if helper.is_inbuilt_cscope() then
 		if vim.loop.fs_stat(M.opts.cscope.db_file) ~= nil then
```

Comment:
This check should be done inside `init_inbuilt_cscope()`
line 390 is good place for it. It'll do other config and skip adding db file since it is not available. 
This will provide flexibility to create db after opening nvim (using `:Cs build`).

Reply:
Yeah, it looks better.